rskj-core/src/main/java/org/ethereum/rpc/LogFilter.java

```diff
@@ -118,28 +117,30 @@ public void newPendingTx(Transaction tx) {
         //empty method
     }
 
-    public static LogFilter fromFilterRequest(FilterRequest fr, Blockchain blockchain, BlocksBloomStore blocksBloomStore) throws Exception {
-        RskAddress[] addresses;
+    public static LogFilter fromFilterRequest(FilterRequest fr, Blockchain blockchain, BlocksBloomStore blocksBloomStore) {
+        return fromFilterRequest(fr, blockchain, blocksBloomStore, 0L, 0L);
+    }
 
+    public static LogFilter fromFilterRequest(FilterRequest fr, Blockchain blockchain, BlocksBloomStore blocksBloomStore, Long maxBlocksToQuery, Long maxBlocksToReturn) {

```

Comment:
```suggestion
    public static LogFilter fromFilterRequest(FilterRequest fr, Blockchain blockchain, BlocksBloomStore blocksBloomStore, long maxBlocksToQuery, long maxBlocksToReturn) {

```

do those need to be `Long`? seems `null` is not being provided there

Reply:
Just in case the config changes and it could provide a null value.